mangaki/mangaki/tests/test_anidb.py

```diff
@@ -4,22 +4,27 @@
 from django.conf import settings
 from django.test import TestCase
 
-from mangaki.models import Editor, Studio
-from mangaki.utils.anidb import client, AniDB
+from mangaki.models import Category, Editor, Studio, Work, Role, Staff, Artist
+from mangaki.utils.anidb import AniDB
 
 
 class AniDBTest(TestCase):
     @staticmethod
+    def create_anime(**kwargs):
+        anime = Category.objects.get(slug='anime')
+        return Work.objects.create(category=anime, **kwargs)
+
+    @staticmethod
     def read_fixture(filename):
-        with open(os.path.join(settings.TEST_DATA_DIR, filename), 'r') as f:
+        with open(os.path.join(settings.TEST_DATA_DIR, filename), 'r', encoding='utf-8') as f:
```

Comment:
Good job, as most of us are Linux / Mac developers, we have `utf-8` as default locale encoding. We wouldn't be able to catch this kind of bug.